code/modules/shuttle/computer.dm

```diff
@@ -366,12 +366,44 @@
 	breakable = FALSE
 	///If true, the lifeboat is in the process of launching, and so the code will not allow another launch.
 	var/launch_initiated = FALSE
+	///If true, the lifeboat is in the process of having the xeno override removed by the pilot.
+	var/override_being_removed = FALSE
+	///How long it takes to unlock the console
+	var/remaining_time = 180 SECONDS 
 
 /obj/structure/machinery/computer/shuttle/lifeboat/attack_hand(mob/user)
 	. = ..()
 	var/obj/docking_port/mobile/crashable/lifeboat/lifeboat = SSshuttle.getShuttle(shuttleId)
 	if(lifeboat.status == LIFEBOAT_LOCKED)
-		to_chat(user, SPAN_WARNING("[src] flickers with error messages."))
+		if(skillcheck(user, SKILL_PILOT, SKILL_PILOT_TRAINED))
+			if(user.action_busy)
+				return
+			else if(!override_being_removed)
```

Comment:
`else if(!override_being_removed)` can just be removed now and everything that was under it tabbed over. In general you should be trying to early return like the above lines are doing rather than if/else chaining when possible.